result/flow.yaml

```diff
@@ -173,17 +159,17 @@ components:
         latency:
           $ref: '#/components/schemas/Metric.Latency'
           x-field-uid: 14
-    Flow.MetricGroup:
+    Flow.MetricTag:
       type: object
       properties:
         name:
           description: |-
-            Name of packet field metric group
+            Name of packet field metric tag
           type: string
           x-field-uid: 1
         value:
           description: |-
-            Value of named packet field metric group
+            Value of named packet field metric tag
           type: string
```

Comment:
In the request, tag value is in hex format. Do we expect here a different format, or it should match with request filter.

Reply:
I think this should always be in hex, (ideally with 0x prepended). The string format, at least, for egress tracking would be incorrect, because we can not assume DUT behaving correctly and there is no 'overlap' of non-tracked bits for the same field and we get unique string values for unique tracked bit combination. May be this will work for ingress tracking; but to keep response format consistent. we should use hex, always.

Reply:
Could not follow , if we have egress tracking set to track all 32 bits of a src ip ( lets assume it is theoritically possible)  , and return src ip in value as 1.1.1.1, 1.1.1.2 etc, ( with Rx for each ) --> How can there be 'overlap' of any bits ? Note I am only talking about use-cases where user has specified tracking on full field ( not on some custom offset or/and length in the packet or in a specific field ) .

Reply:
Tracking on full field, you will get unique strings; but to keep the response format consistent, I think, we should have hex format.